packages/common/src/location/location.ts

```diff
@@ -82,6 +83,11 @@ export class Location {
     });
   }
 
+  ngOnDestroy(): void {
```

Comment:
```suggestion
  /** @nodoc */
  ngOnDestroy(): void {
```
Add nodoc annotation so it doesn't show up on angular.io